app/src/main/sentences/fr/util_yes_no.dslf

```diff
@@ -1,3 +1,3 @@
 util_yes_no: high
-[yes] oui|(pour sûr)|certainement|ok|okay|(Vas y)|affirmatif|procéder|continue|absolument|(fais le)|confirmation;
+[yes] oui|(pour sur)|certainement|(bien sur|entendu)|ok|okay|(vas y)|affirmatif|procéder|continue|absolument|(fais le)|(je? confirm<e|ation>);
```

Comment:
The last block might be capturing too wide : "je confirmation" would be catched. Maybe it would be better written `(je confirme)|confirmation` ?